seeders/20221018204029-seeding-post.js

```diff
@@ -1,16 +1,121 @@
-const posts = require("./posts.json");
-const users = require("./users.json");
-const comments = require("./comments.json");
+// const posts = require("./posts.json");
+const { faker } = require("@faker-js/faker");
+// const comments = require("./comments.json");
+const User = require("../models/index").users;
+
+const users = [];
+const posts = [];
+const comments = [];
+
+async function createUser() {
+  for (let i = 1; i <= 1000; i++) {
+    const user = {
+      email: faker.internet.email(),
+      phone: faker.phone.number("###-###-####"),
+      username: faker.internet.userName(),
+      password: await User.encryptPassword(faker.internet.password()),
+      role: "user",
+      last_edit_date: faker.date.between(
+        "2020-01-01T00:00:00.000Z",
+        "2023-01-01T00:00:00.000Z"
+      ),
+      updatedAt: faker.date.between(
+        "2020-01-01T00:00:00.000Z",
+        "2023-01-01T00:00:00.000Z"
+      ),
+      createdAt: faker.date.between(
+        "2018-01-01T00:00:00.000Z",
+        "2019-12-31T00:00:00.000Z"
+      ),
+    };
+    users.push(user);
+  }
+}
+
+function createPost() {
+  const aspectRatios = [
+    [1280, 720],
+    [500, 500],
+    [800, 600],
+  ];
+  for (let userId = 1; userId <= 1000; userId++) {
+    //generate 0-3 post for each user
+    const numberOfPost = faker.datatype.number({ min: 0, max: 3 });
+    for (let i = 0; i < numberOfPost; i++) {
+      const aspectRatio = faker.helpers.arrayElement(aspectRatios);
+      const width = aspectRatio[0];
+      const height = aspectRatio[1];
+
+      posts.push({
```

Comment:
have you figure how to store the image generated and firebase?

Reply:
I haven't set up the Firebase in the web app but these changes are like a checkpoint so for the next step I will try to upload these generated images to Firebase.

Reply:
Okay, just dont run the seed yet if merged to prod.